### ISSUE TITLE
3.2 - Prompts for notification regardless of settings

### ISSUE THREAD

**Issue by kasperbe** [OneSignal/react-native-onesignal#492]
I have an app, which has been working for a while.

I init OneSignal with following snippet:
```
self.oneSignal = [[RCTOneSignal alloc] initWithLaunchOptions:launchOptions
                                                               appId:@"YOUR_ONESIGNAL_APP_ID"
                                 settings:@{kOSSettingsKeyAutoPrompt: @NO}];
```

This used to work, but since we upgraded to 3.2 it doesn't work anymore.

I've tried using the pure JS version
```
    OneSignal.init('YOUR_ONESIGNAL_APP_ID', {
      kOSSettingsKeyAutoPrompt: "NO"
    });
```

I've tried with 

```
self.oneSignal = [[RCTOneSignal alloc] initWithLaunchOptions:launchOptions
                                                               appId:@"YOUR_ONESIGNAL_APP_ID"
                                 settings:@{kOSSettingsKeyAutoPrompt: @false}];
```

As per the documentation. Nothing really seems to work or have any effect on the prompt.

**Comment by Nightsd01** [comment]
Thanks for posting this issue and we apologize for the inconvenience. 

I've created a PR to resolve this issue (https://github.com/geektimecoil/react-native-onesignal/pull/494)

We expect to have an update released today because this is a relatively major bug.

**Comment by Nightsd01** [comment]
@kasperbe We have released a [new update (3.2.1)](https://github.com/geektimecoil/react-native-onesignal/releases/tag/3.2.1 ) to resolve this problem, thanks for the bug report!

**Comment by beisert1** [comment]
@Nightsd01 I am still seeing this issue on 3.2.5

I've tried both the JS and AppDelegate method of disabling the autoprompt for registering but neither work. The Register for notification prompt is always being automatically displayed at the launch of the app.

Edit: Nevermind, this is working now. I was using react-native-push-notifications for local notifications and had to disable it from there as well.
